app/views/dashboard/users/show.html.erb

```diff
@@ -1,7 +1,18 @@
 <div class="col"
-<h1> <%= @user.first_name %>'s Dashboard </h1>
+<h1 class="name"> <%= @user.first_name %>'s Dashboard </h1>
 
 </div>
 <div class="col">
 <%= link_to 'Create a Trip Log', new_dashboard_trip_log_path, class: "btn btn-primary"%>
 </div>
+
+<% unless @user.destinations.empty? %>
```

Comment:
I will be looking at trying to get a decorator in place to pull these logic statements out.